astropy/io/ascii/tests/test_cds_header_from_readme.py

```diff
@@ -34,22 +32,22 @@ def test_description():
     data = "data/cds/description/table.dat"
     for read_table in (read_table1, read_table2, read_table3):
         table = read_table(readme, data)
-        assert_equal(len(table), 2)
-        assert_equal(table["Cluster"].description, "Cluster name")
-        assert_equal(table["Star"].description, "")
-        assert_equal(table["Wave"].description, "wave ? Wavelength in Angstroms")
-        assert_equal(table["El"].description, "a")
-        assert_equal(
-            table["ion"].description, "- Ionization stage (1 for neutral element)"
+        assert(len(table) == 2)
+        assert(table["Cluster"].description == "Cluster name")
+        assert(table["Star"].description == "")
+        assert(table["Wave"].description == "wave ? Wavelength in Angstroms")
+        assert(table["El"].description == "a")
+        assert(
+            table["ion"].description == "- Ionization stage (1 for neutral element)"
         )
-        assert_equal(
-            table["loggf"].description,
+        assert(
+            table["loggf"].description ==
             "log10 of the gf value - logarithm base 10 of stat. weight times "
             "oscillator strength",
         )
```

Comment:
```suggestion
        assert table["loggf"].description == (
            "log10 of the gf value - logarithm base "
            "oscillator strength"
        )
```
This one is a bit tricky, as the string itself needs to be held together by the parentheses and `black` kept from complaining, but this should work.
Again, `assert` itself should never use the `(...)` as you could see here it just adds potential for confusion with constructs that do need them. In particular they should be removed from all the one-line statements!